src/dashboard/events/cd-dashboard-upcoming-event.vue

```diff
@@ -0,0 +1,216 @@
+          acc1 + session.tickets.filter(ticket => ticket.type === 'ninja').reduce((acc2, ticket) =>
+            acc2 + ticket.quantity, 0), 0);
+      },
+      bookedMentorTickets() {
+        return this.event.sessions.reduce((acc1, session) =>
+          acc1 + session.tickets.filter(ticket => ticket.type === 'mentor').reduce((acc2, ticket) =>
+            acc2 + ticket.approvedApplications, 0), 0);
+      },
+      totalMentorTickets() {
+        return this.event.sessions.reduce((acc1, session) =>
+          acc1 + session.tickets.filter(ticket => ticket.type === 'mentor').reduce((acc2, ticket) =>
+            acc2 + ticket.quantity, 0), 0);
+      },
+      dojoImageUrl() {
+        return `https://s3-eu-west-1.amazonaws.com/zen-dojo-images/${this.dojo.id}`;
+      },
+      dojoImageFallbackImage() {
+        /* eslint-disable global-require */
+        return require('../../assets/avatars/dojo-default-logo.png');
+        /* eslint-enable global-require */
+      },
+      hasOrder() {
+        return this.orders && this.orders.length > 0;
+      },
+      formattedStartTime() {
+        return this.$options.filters.cdTimeFormatter(this.event.dates[0].startTime);
+      },
+      formattedEndTime() {
+        return this.$options.filters.cdTimeFormatter(this.event.dates[0].endTime);
+      },
+      ninjaTickets() {
+        return this.orders.reduce((acc, order) =>
+          acc + order.applications.filter(application => application.ticketType === 'ninja').length, 0);
+      },
+      mentorTickets() {
+        return this.orders.reduce((acc, order) =>
+          acc + order.applications.filter(application => application.ticketType === 'mentor').length, 0);
+      },
+      isChampion() {
+        return this.event.usersDojos.filter(usersDojo => usersDojo.userTypes.indexOf('champion') !== -1).length > 0;
+      },
+    },
+    methods: {
+      async loadOrders() {
+        const res = await EventsService.v3.getOrder(this.loggedInUser.id, {
+          params: {
+            query: { eventId: this.event.id },
+          },
+        });
+        this.orders = res.body.results;
+        this.$emit('loaded');
+      },
+    },
+    watch: {
+      event() {
+        this.loadOrders();
+      },
+    },
+    created() {
+      if (this.event && this.event.id) {
+        this.loadOrders();
+      }
+    },
+  };
+</script>
+
+<style scoped lang="less">
+  @import "~@coderdojo/cd-common/common/_colors";
+  @import "../../common/variables";
+  @import "../../common/styles/cd-filler-loading";
+
+  .cd-dashboard-upcoming-event {
+    margin: 32px 0;
+
+    &__content {
+      background: @cd-white;
+      display: flex;
+      padding: 0 20px;
+    }
+
+    &__main {
+      flex: 2 2 66.6%;
+      padding: 10px 10px 20px 10px;
+    }
+
+    &__dojo {
+      padding: 10px;
+      flex: 1 1 33.3%;
+
+      p {
+        margin: 0;
+      }
+    }
+
+    &__book {
+      display: inline-block;
+      border: 1px solid #0093D5;
```

Comment:
Color è__é

Reply:
Oh yeah, meant to check if we have that in our vars. My bad!